[2023-01-26 18:07:32,471] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Respuesta_Nasa.Respuesta_Confirmacion_NASA scheduled__2023-01-01T00:00:00+00:00 [queued]>
[2023-01-26 18:07:32,488] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Respuesta_Nasa.Respuesta_Confirmacion_NASA scheduled__2023-01-01T00:00:00+00:00 [queued]>
[2023-01-26 18:07:32,489] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-01-26 18:07:32,489] {taskinstance.py:1377} INFO - Starting attempt 10 of 12
[2023-01-26 18:07:32,489] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-01-26 18:07:32,511] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): Respuesta_Confirmacion_NASA> on 2023-01-01 00:00:00+00:00
[2023-01-26 18:07:32,516] {standard_task_runner.py:52} INFO - Started process 863 to run task
[2023-01-26 18:07:32,520] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Respuesta_Nasa', 'Respuesta_Confirmacion_NASA', 'scheduled__2023-01-01T00:00:00+00:00', '--job-id', '647', '--raw', '--subdir', 'DAGS_FOLDER/00-ETL_spaceX.py', '--cfg-path', '/tmp/tmp2bj9gs5r', '--error-file', '/tmp/tmpeif_aywq']
[2023-01-26 18:07:32,521] {standard_task_runner.py:80} INFO - Job 647: Subtask Respuesta_Confirmacion_NASA
[2023-01-26 18:07:32,596] {task_command.py:371} INFO - Running <TaskInstance: Respuesta_Nasa.Respuesta_Confirmacion_NASA scheduled__2023-01-01T00:00:00+00:00 [running]> on host da6de88268cf
[2023-01-26 18:07:32,685] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=frandak2@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Respuesta_Nasa
AIRFLOW_CTX_TASK_ID=Respuesta_Confirmacion_NASA
AIRFLOW_CTX_EXECUTION_DATE=2023-01-01T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=10
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-01-01T00:00:00+00:00
[2023-01-26 18:07:32,687] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-01-26 18:07:32,687] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'sleep 20 && echo "ConfirmaciÃ³n de la NASA, pueden proceder" > /tmp/response_20230101.txt']
[2023-01-26 18:07:32,698] {subprocess.py:85} INFO - Output:
[2023-01-26 18:07:52,703] {subprocess.py:96} INFO - Command exited with return code 0
[2023-01-26 18:07:52,818] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=Respuesta_Nasa, task_id=Respuesta_Confirmacion_NASA, execution_date=20230101T000000, start_date=20230126T180732, end_date=20230126T180752
[2023-01-26 18:07:52,868] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-01-26 18:07:52,897] {dagrun.py:1081} ERROR - Failed to get task '<TaskInstance: Respuesta_Nasa.Notificar_Analistas scheduled__2023-01-01T00:00:00+00:00 [removed]>' for dag 'Respuesta_Nasa'. Marking it as removed.
[2023-01-26 18:07:52,929] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
